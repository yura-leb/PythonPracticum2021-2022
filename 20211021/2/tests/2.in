:sin x sin(x)
sin 3.14
quit всё
