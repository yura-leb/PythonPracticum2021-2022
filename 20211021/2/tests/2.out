0.0015926529164868282
2:3
